docs/getting-started/quick-start.ipynb

```diff
@@ -0,0 +1,203 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quick start\n",
+    "\n",
+    "This section provides a quick introduction to scipp.\n",
+    "For in depth explanations refer to the sections in the user guide."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import scipp as sc\n",
+    "from scipp import Dim"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We start by creating some variables:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var = sc.Variable(dims=[Dim.Y, Dim.X], values=np.random.rand(4,5))\n",
+    "sc.show(var)\n",
+    "print(var)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x = sc.Variable(dims=[Dim.X], values=np.arange(5), unit=sc.units.m)\n",
+    "y = sc.Variable(dims=[Dim.Y], values=np.arange(4), unit=sc.units.m)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We combine the variable into a data array:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "array = sc.DataArray(\n",
+    "    data=var,\n",
+    "    coords={Dim.X: x, Dim.Y: y})\n",
+    "sc.show(array)\n",
+    "print(array)\n",
+    "print(array.values)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Variable can have uncertainties.\n",
+    "Scipp stores these as variances (the square of the standard deviation):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "array.variances = np.square(np.random.rand(4,5))\n",
+    "sc.show(array)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We create a dataset:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset = sc.Dataset(\n",
+    "    {'a': var},\n",
+    "    coords={Dim.X: x, Dim.Y: y},\n",
+    "    labels={'aux' : x})\n",
+    "dataset['b'] = array\n",
+    "dataset['scalar'] = 1.23 * (sc.units.m / sc.units.s)\n",
+    "sc.show(dataset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can slice variable, data arrays, and dataset using a dimension label and an index or a slice object like `i:j`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "dataset['c'] = dataset['b'][Dim.X, 2]\n",
+    "sc.show(dataset)"
+   ]
+  },
+  {
```

Comment:
Add a short comment like "Table representation and plotting is also provided" ?

Reply:
Ok, will do in the next iteration.